src/main/java/com/minecolonies/colony/buildings/AbstractBuilding.java

```diff
@@ -717,13 +752,14 @@ public static View createBuildingView(ColonyView colony, BlockPos id, ByteBuf bu
                     if (c.getName().endsWith("$View"))
                     {
                         Constructor<?> constructor = c.getDeclaredConstructor(ColonyView.class, BlockPos.class);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Local variable 'constructor' could be declared final [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=pmd%3ALocalVariableCouldBeFinal)